toc/rfc/rfc-0040-enhance-stack-handling.md

```diff
@@ -0,0 +1,258 @@
+# Meta
+
+- Name: Provide possibillity to use custom stacks in Cloud Foundry
```

Comment:
The title of this RFC fits better to the [next RFC](https://github.com/cloudfoundry/community/pull/1251) in this context. I think the titles have been swapped by mistake.